Update pages section (#171)
nordcraftengine/documentation: docs/08-pages/01-overview/index.md

---
title: Page fundamentals
description: Explore how pages serve as application foundation with unique URLs, metadata configuration, parameter handling and component containment.
description: Explore how pages work in Nordcraft, providing unique URLs, metadata configuration, parameter handling and component containment.
---

# Page fundamentals
# Pages in Nordcraft

Pages are the foundation of applications in Nordcraft. They represent distinct screens with unique URLs that users navigate to, creating the overall structure of your web application.
Pages are the foundation of applications in Nordcraft, representing distinct screens with unique URLs that users navigate to.

> # What are pages in Nordcraft

Reviewer: Are we sure there are no references to this title?

Author: I found no references
![image](https://github.com/user-attachments/assets/9c6dfaf0-914d-4297-9ff7-9e37f35f3822)

- Connecting to external data through APIs
- Controlling user interactions and navigation flow

An ecommerce site may contain the following pages:

- HomePage, available at `https://your-site.nordcraft.com/`
- About, available at `https://your-site.nordcraft.com/about`
- Products, available at `https://your-site.nordcraft.com/products`
- Checkout, available at `https://your-site.nordcraft.com/checkout`
- etc.

When you create a new project in Nordcraft, a page named `HomePage` is created for you automatically.

## Page structure

A page in Nordcraft can have any of the following:

- [Element tree](/the-editor/element-tree): The hierarchy of elements and components that create the page interface
- [Page configuration](/pages/page-configuration)
  - [URL configuration](/pages/page-configuration#url-structure): Path segments and query parameters that define the page URL on the web
  - [Metadata](/pages/page-configuration#metadata): Title, description, language settings, and social sharing metadata

>   - [Head assets](/pages/page-configuration#head-assets): Custom scripts, styles and other resources, which Nordcraft adds to the page `head` tag

Reviewer: ```suggestion
  - [Head assets](/pages/page-configuration#head-assets): Custom scripts, styles and other resources you want to include in the page's `head` tag
```
